target/linux/mediatek/image/gen_banana_pi_img.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
bash needed here, or can this be done with /bin/sh?

Reply:
I think it does work with POSIX sh, yes. But then I saw a bunch of other invocations of bash in a quick grep, and didn't bother to change it anyway. Have done now.

Reply:
Maybe about half a year ago somebody made an effort to reduce the use of bash in our scripts and convert to sh where easily possible. Some fragments are still found in our patchwork, as not all of the patches have been applied.
I'm mainly following that idea.